src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -173,12 +175,18 @@
     <Compile Include="System\Data\SqlClient\SNI\SSRP.cs" />
     <Compile Include="System\Data\SqlClient\TdsParserStateObjectManaged.cs" />
   </ItemGroup>
+  <!-- Manage the SNI toggle for Windows netstandard and UWP -->
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard' AND '$(TargetsWindows)' == 'true' And '$(IsPartialFacadeAssembly)' != 'true' ">
```

Comment:
I would just remove the part of `'$(IsPartialFacadeAssembly)' != 'true'` since this will always be the case given that we only set it to true for netfx builds.